### PR TITLE
add quick start to readme

### PR DESCRIPTION
**Pull request by dansgithubuser**
I found it hard to put the information contained in this diff together. Did I miss something? In any case, I did this work and I'm offering it to the community.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,6 +37,43 @@ AI tournaments). Changed defaults will be advertised when the match begins.
  - If you would like to develop a bot in Java, please follow [this tutorial](http://sscaitournament.com/index.php?action=tutorial).
  - Want to help improve BWAPI? Fork the [`develop` branch](https://github.com/bwapi/bwapi/tree/develop).
 
+## Quick Start for 4.1.2 {#quick-start-for-4.1.2}
```
Comment (heinermann) [inline README.md]: Don't specify the version. This creates a dependency that developers will have to update for every single release. Just call it "Quick Start".

File: README.md [hunk 2]
```diff
@@ -37,6 +37,43 @@ AI tournaments). Changed defaults will be advertised when the match begins.
...
+1. installation
+  1. install Visual Studio 2013
+  2. install StarCraft
+  3. install Brood War
```
Comment (heinermann) [inline README.md]: You can combine steps 2 and 3 and just say "Install Starcraft: Brood War". The original is an implicit dependency for Brood War. Additionally it's combined in the Battle.net installer anyway.

File: README.md [hunk 3]
```diff
@@ -37,6 +37,43 @@ AI tournaments). Changed defaults will be advertised when the match begins.
...
+  4. go to battle.net, just to get updates, don't need an account
+  5. install BWAPI
+2. compile
```
Comment (heinermann) [inline README.md]: This should be a sub-sections or omitted entirely. Users do **not** need to download BWAPI source code.

File: README.md [hunk 4]
```diff
@@ -37,6 +37,43 @@ AI tournaments). Changed defaults will be advertised when the match begins.
...
+  1. download BWAPI source code
+  2. open bwapi/bwapi.sln
+  3. build ExampleAIModule project, release mode
+  4. copy ExampleAIModule.dll to bwapi-data/AI inside the StarCraft install folder
+3. run StarCraft through Chaoslauncher
+  1. run Chaoslauncher as administrator
```
Comment (heinermann) [inline README.md]: There should be no need to run as administrator.

Comment (Nylithius) [inline README.md]: multiinstance still require admin privileges, otherwise it will crash the game.

**Comment by dansgithubuser** [comment]
Thanks for the quick response! Updated, accepting all suggestions but the admin thing. I tried without admin for single and multi-instance. In both cases you get a popup saying you might need admin priveleges. In multi-instance case, you get a crash. I'd like to leave it in. I think it'd be appropriate to take out when the corresponding issues are fixed.

**Comment by neumond** [comment]
I would like to report some experience from virtualbox without having visual studio (it's simply too heavy to install, also for me its installation simply has failed with windows-style "something gone wrong, sorry, heck you").

1. Installed build tools only http://landinghub.visualstudio.com/visual-cpp-build-tools
2. Run special VC++ related cmd shell from Launch menu.
3. Goto bwapi dir, then `msbuild /p:PlatformToolset=v140 /p:Configuration=Release /p:Platform=Win32`. This will build whole solution. Separate projects can be built by `cd` to project dir and launching same command. In my case I need vs2015 build (to make python3.5 bindings), so I force toolset version via `/p:PlatformToolset=v140`.

You would experience access denied thing if working directly in "program files/BWAPI". I'm doing steps under separately cloned git repo.

**Pull request merged**